test/test_new_big_phi.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# test_new_big_phi.py
+
+import json
+from pyrsistent import v
+
+import pytest
+
+import pyphi
+from pyphi.examples import EXAMPLES
+from pyphi.jsonify import jsonify
+from pyphi.new_big_phi import sia
+from pyphi.compute.subsystem import ces
+from pyphi.relations import relations
+
+NETWORKS = ["basic", "basic_noisy_selfloop", "fig4", "grid3", "xor"]
+
+def expected_sia(example):
+    SIA_PATH = f"test/data/new_big_phi/sia/sia_{example}.json"
+    
+    with open(SIA_PATH) as f:
+        expected = json.load(f)
+    
+    return expected
+
+def expected_ces(example):
+    CES_PATH = f"test/data/new_big_phi/ces/ces_{example}.json"
+    
+    with open(CES_PATH) as f:
+        expected = json.load(f)
+    
+    return expected
+
+def expected_relations(example):
+    RELATIONS_PATH = f"test/data/new_big_phi/relations/relations_{example}.json"
+    
+    with open(RELATIONS_PATH) as f:
+        expected = json.load(f)
+    
+    return expected
+
+def remove_ids(dct: dict):
+    has_id = False
+    
+    for key, value in dct.items():
+        if isinstance(value, dict):
+            remove_ids(value)
+            
+        if isinstance(value, list):
+            for item in value:
+                if isinstance(item, dict):
+                    remove_ids(item)
+            
+        if key == "__id__":
+            has_id = True
+            
+    if has_id:
+        del dct["__id__"]
+
+# Tests
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@pytest.mark.parametrize(
```

Comment:
I think it would be worth factoring out the "convert to JSON, remove IDs, then check equality" pattern that's common to all these tests.